src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -523,15 +523,18 @@ def __load_status_file_components(self, initial_load=False):
         for i in range(0, len(status_file_data['status']['substatus'])):
             name = status_file_data['status']['substatus'][i]['name']
             if name == Constants.PATCH_INSTALLATION_SUMMARY:     # if it exists, it must be to spec, or an exception will get thrown
-                message = status_file_data['status']['substatus'][i]['formattedMessage']['message']
-                self.__installation_summary_json = json.loads(message)
-                self.__installation_packages = self.__installation_summary_json['patches']
-                self.__maintenance_window_exceeded = bool(self.__installation_summary_json['maintenanceWindowExceeded'])
-                self.__installation_reboot_status = self.__installation_summary_json['rebootStatus']
-                errors = self.__installation_summary_json['errors']
-                if errors is not None and errors['details'] is not None:
-                    self.__installation_errors = errors['details']
-                    self.__installation_total_error_count = self.__get_total_error_count_from_prev_status(errors['message'])
+                if self.execution_config.exec_auto_assess_only:
+                    self.__installation_substatus_json = status_file_data['status']['substatus'][i]
+                else:
+                    message = status_file_data['status']['substatus'][i]['formattedMessage']['message']
+                    self.__installation_summary_json = json.loads(message)
+                    self.__installation_packages = self.__installation_summary_json['patches']
```

Comment:
No, the else part here is used to preserve details, for non Auto Assessment operations, whenever and whichever properties, where required. For eg: on a reboot, that happens in between an Installation operation, we need to preserve previous state data such as any error details, reboot status, etc in PatchInstallationSummary, everything under PatchAssessmentSummary (which will be overwritten by implicit Assessment later), everything under ConfigurePatchSummary and so on. 

If part was added since we need to preserve all contents within these summaries from the previous iteration for the same seq.